### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1][WIP] Search results not conveyed by using input-group

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11255]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/core/input-group
Using input-group we have implemented search functionality of List. Its working fine. Able to search but screen reader not able to read searched results.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

#### Is there anything else we should know?
Refer Attachement to see code used
<img width="1728" alt="Screenshot 2024-01-21 at 3 46 45 PM" src="https://github.com/SAP/fundamental-ngx/assets/157268666/e384118c-96b7-44c1-9bc0-45dacc20926f">

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

Here the reporter expects to have an announcement of the available results after search.
In order to have this, you need to provide a service to send messages to the screen reader (in UI5 called InvisibleMessage), which the application can use to announce the results.

@JyotiKumari1507  - Is there a formal acc requirement to have this? To my understanding the change in the UI happens after the current focus position, so the user can go down and check the result of the search and then back on the search to refine further. Although it looks not so usable and comfortable, I think this is still considered accessible and the request here can rather be considered as a usability enhancement? Or I am missing something?

Thanks,
Nikolay

**Comment by JyotiKumari1507** [comment]
@nikolay-kolarov  @droshev  As per accessibility team, Search results should be conveyed to the user.. On entering any value in the search field and pressing enter key, list get filtered based on the entered value but the same is not conveyed to the screen reader users. screenshot attached for references
<img width="903" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/bfb43470-31f3-448d-96cc-f993b0918831">

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev please let us know what fields are pending to fill from our side

**Comment by droshev** [comment]
@JyotiKumari1507 did you set all aria attributes needed? If so, can you create a small stackblitz example demonstrating that you cannot set this on the application side?

**Comment by JyotiKumari1507** [comment]
@divyamalhotra488 please share stackblitz example

**Comment by droshev** [comment]
@JyotiKumari1507 any update?

**Comment by divyamalhotra488** [comment]
Hi @droshev, please find stackblitz example here: https://stackblitz.com/edit/eclhkf-bgsdzx?file=src%2Fapp%2Flist-navigation-example.component.ts

**Comment by mikerodonnell89** [comment]
> Hi @droshev, please find stackblitz example here: https://stackblitz.com/edit/eclhkf-bgsdzx?file=src%2Fapp%2Flist-navigation-example.component.ts

These two components (input and list) are their own standalone components, unaware of each other. This should be handled on the application side
